core/db_models/EEM_Payment_Method.model.php

```diff
@@ -360,23 +452,26 @@ public function get_all_for_transaction($transaction, $scope)
 
     /**
      * Returns the payment method used for the last payment made for a registration.
+     * Note: if an offline payment method was selected on the related transaction then this will have no payment
+     * methods returned. It will ONLY return a payment method for a PAYMENT recorded against the registration.
      *
-     * Note: if an offline payment method was selected on the related transaction then this will have no payment methods returned.
-     * It will ONLY return a payment method for a PAYMENT recorded against the registration.
-     *
-     * @param EE_Registration|int $registration_or_reg_id  Either the EE_Registration object or the id for the registration.
+     * @param EE_Registration|int $registration_or_reg_id Either the EE_Registration object or the id for the
+     *                                                    registration.
      * @return EE_Payment|null
+     * @throws EE_Error
+     * @throws EE_Error
+     * @throws EE_Error
```

Comment:
Same ?

Reply:
weird... never happened in the past (that I'm aware of), so must have just started after an update